working/2509-private-named-parameters/feature-specification.md

```diff
@@ -395,9 +396,11 @@ with private name *p* in constructor C:
     compile-time error. *If removing the `_` leads to a collision with
     another parameter, then there is a conflict.*
 
-*   Otherwise, the name of the parameter in C is *n*. *If the parameter is
-    named, this then avoids the compile-time error that would otherwise be
-    reported for a private named parameter.*
+*   Otherwise, the name of the parameter in the signature of C is *n*. By "in
```

Comment:
The text above says
> given a named initializing formal or declaring parameter …

I want this feature to work for positional parameters too.
It's not a compile-time error to write `C(this_foo)`, but I still rewrite it as `C(int foo) : _foo = foo;` to avoid having a private name in the public API and docs.

I don't want to have to make my parameters named to get this feature.

Reply:
I don't think it makes sense for positional parameters to be part of this feature because "the parameter name at the callsite" isn't meaningful to the language when it comes to positional parameters.

The language already allows a positional parameter to have a private name and this proposal doesn't change that. I'll make at note in the non-normative dartdoc section to suggest they handle positional parameters with private names in the same way.